openest/generate/daily.py

```diff
@@ -97,7 +97,7 @@ def format(self, lang):
 
     def apply(self, region, *args):
         def generate(region, year, temps, **kw):
-            if isinstance(curvegen, CurveGenerator):
+            if isinstance(self.spline, CurveGenerator):
```

Comment:
This won't work. It should be `model` rather than `self.spline`, but then we need to use `get_curve` below, rather than `get_spline`. I'm not convinced this is such a good use of your time. Probably better to move old code to something like "unmaintained.py", so we don't lose it, but don't bother with it until we need it.

Reply:
I would just as soon delete it in a commit and someone can go look at version control for the old version, if they need it. Maybe can replace it was an `UnimplementedError` or something.

Reply:
@jrising What do you suggest we mothball? The `apply()` or this entire subfunction?

Reply:
The whole of `YearlyDayBins`. But I don't want to delete it, because we will want this kind of functionality. When you're more comfortable with the classes that do work, and how they work, you can fix this.